Sources/ArcGISToolkit/Components/Authentication/LoginViewModifier.swift

```diff
@@ -0,0 +1,98 @@
+// Copyright 2022 Esri.
```

Comment:
```suggestion
// Copyright 2023 Esri.
```

Reply:
I didn't change it originally because it is still the original `Login.swift` file. But since it has been renamed and significantly changed, I suppose it is best to treat it as a new file.